src/ValueComponent.js

```diff
@@ -76,6 +76,7 @@ let DragDropSpec = {
         object: null,
         itemid: ""
       });
+      if (!qlik.currApp()) return;
```

Comment:
So if this is not present in export it will never work to export the multi KPI including a visualization right?

Reply:
so as I tested in windows PTE, when qlik.currApp() is null, download story as pdf still works.